routers/repo/commit.go

```diff
@@ -246,6 +247,15 @@ func Diff(ctx *context.Context) {
 	ctx.Data["Parents"] = parents
 	ctx.Data["DiffNotAvailable"] = diff.NumFiles() == 0
 	ctx.Data["SourcePath"] = setting.AppSubURL + "/" + path.Join(userName, repoName, "src", "commit", commitID)
+
+	note := git.Note{}
+	err = git.GetNote(ctx.Repo.GitRepo, commitID, &note)
```

Comment:
```suggestion
	err = git.GetNote(ctx.Repo.GitRepo, commitID, note)
```

Reply:
Thanks. I'm not familiar with Go and this is something that would not be allowed in other languages (taking the address of an rvalue).